force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -271,11 +336,18 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static Integer findWordInList(String wordToFind, List<String> words) {
         // The variable to store the index of the word
-
+        list <String> matchFound = new List<String>();
```

Comment:
Try to remove variables that are not used for anything, sometimes they confuse people